### ISSUE TITLE
Background notifications with mutable-content flag (iOS)

### ISSUE THREAD

**Issue by artdevgame** [wix/react-native-notifications#848]
Hello 👋🏻 

I'm using this plugin to consume push notifications dispatched by the Twilio Conversation API.

I have wired up my AppDelegate as instructed and it includes the implementation of the `didReceiveRemoteNotification` method, however, it never seems to get triggered (I added a NSLog to try and debug and no output occurred)

This is an example payload that Twilio is sending (I have no control over the actual structure):
```
{
    aps =     {
        alert =         {
            body = "[{\"type\":\"paragraph\",\"children\":[{\"text\":\"yo\"}]}]";
        };
        "mutable-content" = 1;
    };
    author = "045ce068822a13168def8943fce894b38e4e1968";
    "conversation_sid" = CHc7ab9d45c3774796940931887f4c0226;
    "conversation_title" = "Test convo";
    "message_index" = 40;
    "message_sid" = IM04c2c8eae6824a29a6dead82eb64427b;
    "twi_message_id" = RUdd90d3549e57dcdabb0484a7564f70c0;
    "twi_message_type" = "twilio.conversations.new_message";
}
```

I've been able to intercept the push with a 'notification service extension' but ideally I'd like to hook into the `registerNotificationReceivedBackground` function in JS, which isn't executed at the moment.

Should the Twilio payload in its current form cause `didReceiveBackgroundNotification` to execute or not? If not, is there something I can do inside the service extension to make it execute?

**Comment by artdevgame** [comment]
@DanielEliraz Sorry to ping you directly on this one, but you seem to be the only active contributor and I'm looking for an answer on the above, even if it's to say 'that won't work' kind of thing. Do you happen to know the answer?

**Comment by DanielEliraz** [comment]
No problem @artdevgame. Can you make Twilio send it without the `alert` entry in the `aps` object. This is what Apple [demands](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/pushing_background_updates_to_your_app?language=objc#:~:text=To%20send%20a%20background%20notification%2C%20create%20a%20remote%20notification%20with%20an%20aps%20dictionary%20that%20includes%20only%20the%20content%2Davailable%20key%2C%20as%20shown%20in%20Listing%201.)

**Comment by artdevgame** [comment]
> No problem @artdevgame. Can you make Twilio send it without the `alert` entry in the `aps` object. This is what Apple [demands](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/pushing_background_updates_to_your_app?language=objc#:~:text=To%20send%20a%20background%20notification%2C%20create%20a%20remote%20notification%20with%20an%20aps%20dictionary%20that%20includes%20only%20the%20content%2Davailable%20key%2C%20as%20shown%20in%20Listing%201.)

No, unfortunately. I have no control over that payload at all 😢 , though I'll forward that link on to their tech support and see what they say. Thanks.
